seccomp.go

```diff
@@ -730,6 +734,23 @@ func (f *ScmpFilter) GetNoNewPrivsBit() (bool, error) {
 	return true, nil
 }
 
+// GetLogBit returns the current state the Log bit will be set to on the filter
+// being loaded, or an error if an issue was encountered retrieving the value.
+// The Log bit tells the kernel that all actions taken by the filter, with the
+// exception of ActAllow, should be logged.
```

Comment:
Can you add something in the comment regarding a minimum required kernel version for this working?

Reply:
Yes, this is a good idea. There's, unfortunately, now a little uncertainty around which kernel release it'll be present in due to an unrelated [issue](https://lkml.kernel.org/r/<alpine.LRH.2.21.1709101406320.22614@namei.org>). I'll update the comment as soon as there's some clarity.

Reply:
In the update to this PR, I've added a comment about the minimum API level required (3).